lib/tether/ops_linux.go

```diff
@@ -699,6 +700,52 @@ func (t *BaseOperations) MountTarget(ctx context.Context, source url.URL, target
 	return nil
 }
 
+// CopyExistingContent copies the underlying files shadowed by a mount on a directory
+// to the volume mounted on the directory
+// see bug https://github.com/vmware/vic/issues/3482
+func (t *BaseOperations) CopyExistingContent(source string) error {
+	defer trace.End(trace.Begin(fmt.Sprintf("copyExistingContent from %s", source)))
+
+	source = filepath.Clean(source)
+
```

Comment:
Before creating directories and `bind` mounting the underlying mount point, it may be worth checking if there are even files there.  There's no sense descending down the mount/copy path if there's nothing to copy, right?